mdfreader/mdf3reader.py

```diff
@@ -1312,7 +1330,7 @@ def writePointer(f, pointer, value):
                 pointers['CN'][dataGroup][channel]['longChannelName'] = fid.tell()
                 fid.write(pack(LINK, 0))  # pointer to long channel name
                 fid.write(pack(LINK, 0))  # pointer to channel display name
-                fid.write(pack(UINT16, 0))  # No Byte offset
+                fid.write(pack(UINT16, byteOffset))  # No Byte offset
```

Comment:
Probably want to remove the comment that says "# No Byte offset here" that was lazy on my part.  :)